### DISCUSSION TITLE
Idea: Create a discord group

### DISCUSSION THREAD

**Discussion by sherkhan30452** [friction2d/friction#111] [Ideas]
Well most End-User doesn't have an github accounts as it is meant for computer programmers or developers. I suggest to create a Discord group since it is most popular among many users as well used by most FOSS developers to get feedback at one place.

**Comment by rodlie** [comment]
Yeah... It was something I considered, but it's another place I need to monitor/manage. It will also fragment stuff, as most people will use Discord to submit issues etc, and they (the issue/info/feedback/whatever) will not end up on GitHub (where I want it).

But, if this is something people want I can of course do something about it, but don't expect me to be active there.

**Reply by sherkhan30452** [reply]
Yes, I totally agree with you. Most people share issues (in message form) and leave it to lead developer to manually create issue on github.

**Reply by rodlie** [reply]
When the community get large enough (I don't know how many users there are) we can setup a Discord, but we need one or two persons that are willing to moderate the group (I only login to Discord one or two times a day, if at all).

**Reply by doc-halo** [reply]
There's a subreddit with only 5 people. So maybe direct people there for now.

**Reply by sherkhan30452** [reply]
> There's a subreddit with only 5 people. So maybe direct people there for now.

Can you please share link to that reddit group.

**Reply by doc-halo** [reply]
https://www.reddit.com/r/frictiongraphics/

**Comment by dejayc** [comment]
I like the idea of being able to get real-time-ish feedback about new feature ideas, coding implementations, etc.

**Reply by dejayc** [reply]
UPDATE: I wonder if people would be supportive of having an unofficial "Friction User Group" Discord server to collaborate on sharing ideas and tips and tricks for Friction.

**Reply by brendanhowell** [reply]
I really like the initiative but I have some reservations about Discord.

I know Discord is very popular in certain groups but it's an extremely proprietary thing that is controlled by one company. With any platform service it's just a question of when, not if, they will change the terms of service and screw you over. If you put tons of special knowledge and centralize the community there you will be entirely dependent on the good will of a bunch of business guys who want to get rich. In the long run, you will regret it.

Using a proprietary service to support a free/open source project is also a bit inconsistent to the values of the whole endeavor. To make a comparison - imagine a organic food activists meeting at McDonalds, consumer activists meeting at Walmart or e-waste activists meeting at the Apple store. I realize that not all users identify as "activists" but I hope you see my point.

Also, Discord is not so well loved among people over 30. I don't say this as a grumpy-old-man complaint.  I mean to say that the people who do use Discord services tend to skew towards a very specific demographic, so if you have that as the center of your community it will inadvertently exclude a lot of people who don't get discord or have the spoons to get into it.

The idea of having a place for community exchange is a great one but I think it would be better served by a different service.

**Reply by rodlie** [reply]
I have no problems with Discord in general, but it's not a good place to keep or find information. I also don't like fragmentation, projects like this should have a unified place for community communication etc.

---
_Regarding the proprietary stuff, I don't care. Friction is already hosted on GitHub, a proprietary service. Mirrored on SourceForge, another proprietary service. Built on Windows, a proprietary platform using a proprietary compiler........ You get my point._

**Reply by rodlie** [reply]
Yeah, let's not go down the freedom rabbit hole :smile: 

I don't care about stuff been free as in freedom. Does the thing work? cool, move on. If the thing does not work anymore or they do something stupid that affects the project, move to something else, or host it. Until that time happens I don't have time to think about stuff the may happen in theory.

At the end of the day Friction is just some git repos and a website, we don't care about the underlying infrastructure, I can drop it within an hour and be up and running on something else.

**Comment by dejayc** [comment]
> I have no problems with Discord in general, but it's not a good place to keep or find information. I also don't like fragmentation, projects like this should have a unified place for community communication etc.

This is true.  I also believe that not every utterance of communication needs to be memorialized.  Sometimes people can have a long conversation that ends up turning into one or two small bullet points in a formal document.  In this case, a more informal avenue of communication can be useful for facilitating collaboration.

**Reply by rodlie** [reply]
btw, I could probably contact https://discuss.pixls.us and get a dedicated category for Friction, they are usually very open to host new apps (related to graphics), I'm part of the Natron dev team and we have used pixls as our primary forum for years.

Other than that, I don't know. GitHub works for me as a developer, but I understand that regular users may have problems with GitHub as a "forum" or anything.

**Comment by dejayc** [comment]
> GitHub works for me as a developer, but I understand that regular users may have problems with GitHub as a "forum" or anything.

Maybe this product is too new to benefit from multiple channels of communication, but projects can often benefit from having different channels for different purposes.

Forums are a very "old-school", slow way of communicating, which hopes that memorializing random conversations forever will have some value.  Sites and apps like Discord favor rapid, real-time capable communication that introduces more noise in the long-term, for sure, but also facilitates faster and more interactive collaboration.

Personally, I find value in having diverse communication channels that serve different purposes:

- A site, like featureupvote.com, to allow users to propose and vote upon product ideas and roadmap
- A site, like stackoverflow.com, to allow users to submit questions and answers
- A wiki to host editorially-curated documentation for end users
- A forum to facilitate random conversations that may (but often not) add some searchable value if recorded indefinitely
- A bug-tracking or issue-raising forum for documenting point-in-time issues that need to be fixed
- A real-time communication channel like Discord or Slack, to allow collaborators to iterate through ideas quickly and without expectation of being recorded forever

I mean, to be truthful, all of the purposes above could be accomplished by a newsgroup or email list, but there's a reason that no one takes that approach.
